qiling/debugger/gdb/gdb.py

```diff
@@ -444,21 +444,20 @@ def handle_q(subcmd: str) -> Reply:
                     ]
 
                 # os dependent features
-                if not self.ql.interpreter:
+                features += [
+                    'QEnvironmentHexEncoded+',
+                    'QEnvironmentReset+',
+                    'QEnvironmentUnset+'
+                ]
+
+                # filesystem dependent features
+                if hasattr(self.ql.os, 'path'):
                     features += [
-                        'QEnvironmentHexEncoded+',
-                        'QEnvironmentReset+',
-                        'QEnvironmentUnset+'
+                        'QSetWorkingDir+',
+                        'qXfer:auxv:read+',
+                        'qXfer:exec-file:read+'
                     ]
 
-                    # filesystem dependent features
-                    if hasattr(self.ql.os, 'path'):
-                        features += [
-                            'QSetWorkingDir+',
-                            'qXfer:auxv:read+',
-                            'qXfer:exec-file:read+'
-                        ]
-
                     # process dependent features
                     if hasattr(self.ql.os, 'pid'):
```

Comment:
That one was also under `if not self.ql.interpreter`, so it should be re-indented appropairately.